packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -514,15 +664,17 @@ function arePythonToolsInstalled(): boolean {
 
 // Since TypeScript can only get the CLI tool's error output as a string, we have to parse and sanitize it ourselves
 function parseCliErrorMessage(message: string): string {
-    const errorMatch = message.match(/ERROR: .*/)
-    const botoMatch = message.match(/(?<=botocore\.exceptions\.).*/) // Boto errors have a different match
+    const cfnMatch = message.match(/ERROR: .*/)
+    const botoMatch = message.match(/(?<=botocore\.exceptions\.).*/) // Boto errors have a special match
     const terraformMatch = message.match(/AttributeError:.*/) // Terraform CLI responds with a different error schema... this catches invalid .json plans
-    if (errorMatch?.[0]) {
-        return errorMatch[0]
+    if (
+        message.includes('The security token included in the request is invalid') ||
+        message.includes('The security token included in the request is expired')
```

Comment:
nit: same comment as before, try to prefer comparing to an exception type if possible rather than the message.